RobotsTxtParser.php

```diff
@@ -255,14 +264,15 @@ public function prepareRules()
 	 */
 	private function shouldSwitchToZeroPoint()
 	{
-		return in_array(strtolower($this->current_word), array(
+		return in_array(strtolower($this->current_word, array(
```

Comment:
Please, close ")" after $this->current_word. There an error according to phpunit tests:
```
Parse error: syntax error, unexpected ';' in /home/travis/build/bopoda/robots-txt-parser/RobotsTxtParser.php on line 276
```